bundles/org.openhab.binding.androiddebugbridge/README.md

```diff
@@ -20,9 +20,11 @@ Please update this document if you tested it with other android versions to refl
 
 ## Discovery
 
-As I can not find a way to identify android devices in the network the discovery will try to connect through adb to all the reachable ip in the defined range.
+Android TV and Fire TV devices should be discovered automatically (using mDNS).
 
-You could customize the discovery process through the binding options. 
+As I can not find other android a way to identify other android devices in the network the manual discovery scan will try to connect through adb to all the reachable ip in the defined range.
```

Comment:
Please update this line to somehting like: The binding is unable to....

Reply:
Text updated!